### ISSUE TITLE
HomeKit says no response after power outage

### ISSUE THREAD

**Issue by kinergy** [evgolsh/homebridge-crestron-home#3]
**Describe The Bug:**
When my home loses power unexpectedly something gets messed up after bootup. All the lights in Apple's Home app lose connection and say "No Response". Restarting Homebridge or this Homebridge plugin doesn't help. The Homebridge plugin maintains connectivity to Crestron because I can turn lights on/off in the web UI. The only solution I have found is to remove the bridge from Apple Home and then rescan the QR code and add everything back in. Doing this for about 120 lights and scenes is really painful...

I also use another Homebridge plugin connecting Ubiquiti cameras, that plugin doesn't lose connectivity to Apple Home during an unexpected power cycle.

**To Reproduce:**
Power cycle everything at once

**Expected behavior:**
After an unexpected power cycle, I'd expect the lights to continue working in the Apple Home app.

**Logs:**

```
Show the Homebridge logs here, remove any sensitive information.
```

**Plugin Config:**

```json
{
    "bridge": {
        "name": "Homebridge E5B2",
        "username": "<deleted>",
        "port": 51560,
        "pin": "<deleted>",
        "advertiser": "ciao"
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "auth": "form",
            "theme": "auto",
            "tempUnits": "f",
            "lang": "auto",
            "platform": "config"
        },
        {
            "platform": "UniFi Protect",
            "_bridge": {
                "username": "<deleted>",
                "port": 34330
            },
            "controllers": [
                {
                    "address": "192.168.1.1",
                    "username": "<deleted>",
                    "password": "<deleted>"
                }
            ]
        },
        {
            "name": "Crestron Home Platform",
            "crestronHost": "192.168.1.196",
            "token": "<deleted>",
            "enabledTypes": [
                "Switch",
                "Dimmer",
                "Scene"
            ],
            "updateInterval": 30,
            "platform": "CrestronHomePlatform",
            "_bridge": {
                "username": "<deleted>",
                "port": 46599
            }
        }
    ],
    "disabledPlugins": []
}
```

**Screenshots:**

**Environment:**

* **Plugin Version**: 1.1.4
* **Homebridge Version**: 1.6.0
* **Node.js Version**: 18.12.1
* **NPM Version**: 
* **Operating System**: Raspbian

**Comment by evgolsh** [comment]
Hello,

From time to time I experience the same behavior. I didn't find the root cause yet because it happens very rarely. 

You don't need to remove the bridge, deleting cached accessories will help.
In my case (running Homebridge on Raspbian):

Ssh to the Raspberry Pi
`cd /var/lib/homebridge/accessories/`
`sudo hb-service stop`
`sudo rm cachedAccessories.[YOUR-BRIDGE]`
`sudo hb-service start`

The Homekit should work properly after this. I'll try to debug the issue since it happens not only to me.

**Comment by kinergy** [comment]
That worked, thanks! Would be amazing if there was a fix for this - I bought a battery backup to prevent the RPi from power cycling but the recent storms and power outages in CA drained that as well :)

**Comment by evgolsh** [comment]
Trying to debug it. Are you using Apple TV as a hub for the HomeKit?

**Comment by kinergy** [comment]
I have two Apple TVs and two HomePods. I've noticed that the hub tends to end up on a HomePod consistently with the other devices as backups.

**Comment by evgolsh** [comment]
Please try 1.1.5, I made a few fixes, see if it helps

**Comment by evgolsh** [comment]
fixed in 1.1.8
